doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -20,20 +20,168 @@
+| `req.gen.rsl.01` | Resiliency | The Architecture **must** support resilient Kubernetes components that are required for the continued availability of running workloads. |
+| `req.gen.rsl.02` | Resiliency | The Architecture **should** support resilient Kubernetes service components that are not subject to `req.gen.rsl.01`. |
+| `req.gen.avl.01` | Availability | The Architecture **must** provide High Availability for Kubernetes components. |
+| `req.gen.ref.01` | Model | The Architecture **must** support the Reference Model defined profiles (instance types: Basic, Network Intensive, Compute Intensive). |
+
+
+<p align="center"><b>Table 2-1:</b> Kubernetes Architecture: General Requirements</p>
+
+<a name="2.3.2"></a>
+### 2.3.2 Infrastructure
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|------------------------------------------|
+| `req.inf.com.01` | Compute | The Architecture **must** provide compute resources for Pods.  |
+<!--
+| `req.inf.com.02` | Compute | The Architecture **should** include industry standard hardware management systems at both HW device level (embedded) and HW platform level (external to device). |
+-->
+| `req.inf.com.03` | Network | The Architecture **must** support Container Runtime Interface (CRI). |
+|----|------------------------------------------------|----------------------------------------|
+| `req.inf.stg.01` | Storage | The Architecture **must**  support for Kubernetes Volumes for container storage.
+| `req.inf.stg.02` | Storage | The Architecture **may** provide shared Object storage as a service for Containers workload.
+| `req.inf.stg.03` | Storage | The Architecture **must** support Container Storage Interfaces (CSI).
+| `req.inf.stg.04` | Storage | The Architecture **may** support Software Defined Storage (SDS) that seamlessly supports shared block storage, object storage and flat files. |
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable storage. |
+| `req.inf.stg.06` | Storage | The Architecture **must** support ephemeral storage (non-persistent) storage for Pods. |
+| `req.inf.stg.07` | Storage | The Architecture **must** support persistent storage for Pods. |
+| `req.inf.ntw.01` | Network | The Architecture **must** support Conatiner Network Interface (CNI). |
+| `req.inf.ntw.02` | Network | The Architecture **must** support intra-node communications, such as between agents on a node and all pods on that node |
+| `req.inf.ntw.03` | Network | The Architecture **must** support inter-node communications without NAT, such as communications between pods on a node with all other pods on all nodes |
+| `req.inf.ntw.04` | Network | The Architecture **must** support low latency and high throughput traffic needs. |
+| `req.inf.ntw.05` | Network | The Architecture **should** support service function chaining. |
```

Comment:
Here I believe we need to have a separate discussion as there is no multi-vendor interface today between the NFVI and the CNFs for service chaining. Requiring support with no means would mean that the requirement cannot be effectively verified.